scripts/variantstore/wdl/MergePgenHierarchical.wdl

```diff
@@ -145,6 +149,8 @@ task MergePgen {
         disks: "local-disk ${disk_in_gb} HDD"
         bootDiskSizeGb: 15
         cpu: "${cpu}"
+        # The PGEN Docker needs to be in GAR or GCR to be able to use noAddress
+        # noAddress: true
```

Comment:
Why is this here if it's commented out?

Reply:
I was intending to leave behind a breadcrumb so that when VS-1315 (make a PLINK Docker and put it in GCR) is implemented, the author would go back and turn on `noAddress` here. But now that you mention it the VS-1315 is probably a better place for that breadcrumb.